.gitignore

```diff
@@ -23,6 +23,7 @@ __pycache__
 venv*/
 .venv*/
 opentelemetry-python-core*/
+/opentelemetry-python-core
```

Comment:
What's the difference between this and above? Maybe they can be combined

Reply:
I used symlink instead of checking out another copy of core which did not work with the existing rule. Probably could be combined but I didn't understand fully how others checkout the repo so didn't want to break anyone's flow.

Reply:
might be better to move this to a .gitignore_global, if it's user-flow related.

But either not way not really worried about slightly bloating the gitignore. It may be good to add a comment to clarify this is for local dev scenarios.